wimprates/migdal.py

```diff
@@ -52,7 +52,7 @@ def read_migdal_transitions(material='Xe'):
 
 def _default_shells(material):
     consider_shells = dict(
-        Xe=['3*', '4*', '5_0'],
```

Comment:
@JelleAalbers, I think for the general wimprates environments, you don't want to have this. So I'm reverting this local change